### PR TITLE
#170947431 deploy base app on heroku

### PR DESCRIPTION
**Pull request by izzett222**
#### What does this PR do?
Deploy the base application to Heroku
#### Description of Task to be completed?
- add the procfile
- create Heroku staging and production app
- add build scripts in package.json
#### How should this be manually tested?
- open [staging app link](https://octopus-bn-backend-staging.herokuapp.com/) or [production app link](https://octopus-bn-backend.herokuapp.com/)
#### What are the relevant pivotal tracker stories?
#170947431
#### Questions:

### CODE REVIEW

**Comment by blaise82** [comment]
is it possible that you may add a link to our Heroku app
or even a screenshot if possible

File: procfile [hunk 1]
```diff
@@ -0,0 +1 @@
+web: npm run build
```
Comment (AbdoulNuru) [inline procfile]: @izzett222 Eslint

Comment (izzett222) [reply procfile]: when I add a newline , its give it still show the error

Comment (izzett222) [reply procfile]: @AbdoulNuru I fixed the problem

**Review by hakoemmy — APPROVED** [review]

**Review by AbdoulNuru — APPROVED** [review]

**Review by blaise82 — APPROVED** [review]

**Review by kcharles52 — APPROVED** [review]

**Pull request merged**